src/dbcp/etl.py

```diff
@@ -31,8 +31,8 @@ def etl_eip_infrastructure() -> Dict[str, pd.DataFrame]:
 
 def etl_lbnl_iso_queue() -> Dict[str, pd.DataFrame]:
     """LBNL ISO Queues ETL."""
-    source_path = Path("/app/data/raw/queues_2022_clean_data.xlsx")
-    lbnl_raw_dfs = dbcp.extract.lbnl_iso_queue.extract(source_path)
+    lbnl_uri = "gs://dgm-archive/lbnl_iso_queue/queues_2023_clean_data.xlsx"
```

Comment:
Using the `gs://dgm-archive` bucket instead of github for consistency. Planning on migrating everything from github to this bucket at some point.